packages/gatsby-theme-newrelic/src/components/SearchInput.js

```diff
@@ -118,6 +119,14 @@ const SearchInput = forwardRef(
               border: 1px solid rgba(0, 126, 138, 0.6);
               box-shadow: 0 0 0 4px rgba(0, 126, 138, 0.1);
             }
+
+            &:hover {
+              border: 1px solid var(--border-color);
```

Comment:
```suggestion
              border-color: var(--border-color);
```

Assuming the change is made above to use a transparent border color, you can instead just update the border color here.